cmd/syft/cli/options/cataloger.go

```diff
@@ -0,0 +1,27 @@
+package options
+
+import (
+	"fmt"
+
+	"github.com/anchore/clio"
+	"github.com/anchore/syft/syft/source"
+)
+
+type Cataloger struct {
```

Comment:
should this remain unexported since it's meant for only other options here?

Reply:
yes, except it conflicts with the syft `cataloger` package. very open to naming suggestions here...